_layouts/page.html

```diff
@@ -6,7 +6,7 @@
     <div class="container">
         <div class="logo">
             <a href="/">
-                <img src="/assets/img/logo.svg"/>
+                <img class="w-100" src="/assets/img/mlh-prep.png"/>
```

Comment:
Having `alt` attribute in the `img` tags would be great in terms of accessibility. Also, to have better [CLS](https://web.dev/cls/), it is a good practice to add `width` and `height` attributes to the images in the DOM.

Reply:
Awesome 🙌👍